extractor/src/main/java/org/schabi/newpipe/extractor/stream/StreamExtractor.java

```diff
@@ -178,6 +178,39 @@ public StreamExtractor(StreamingService service, LinkHandler linkHandler) {
     @Nonnull
     public abstract String getUploaderAvatarUrl() throws ParsingException;
 
+    /**
+     * The Url to the page of the parent chanel of the stream. This must not be a homepage,
+     * but the page offered by the service the extractor handles. This url will be handled by the
+     * <a href="https://teamnewpipe.github.io/documentation/03_Implement_a_service/#channel">ChannelExtractor</a>,
```

Comment:
Use `{@link ChannelExtractor}` instead

Reply:
hmm, truth is I just copied the correspond comment from `getUploaderUrl`. How can I use `{@link ChannelExtractor}`?

Reply:
@Royosef: Just change the whole `<a href="https://teamnewpipe.github.io/documentation/03_Implement_a_service/#channel">ChannelExtractor</a>` to `{@link ChannelExtractor}`.